js/interface.js

```diff
@@ -595,8 +619,127 @@ function activateFind() {
   }
 }
 
-function sortDataSources(key, order) {
-  var toBeOrderedDataSources = dataSources;
+function restoreItem(id, name) {
+  Fliplet.API.request({
+    url: 'v1/data-sources/' + id + '/restore',
+    method: 'POST'
+  }).then(function() {
+    $('.data-source[data-id="' + id + '"]').remove();
+
+    trashSources = trashSources.filter(function(ds) {
+      return ds.id !== id;
+    });
+
+    Fliplet.Modal.alert({
+      title: 'Restore complete',
+      message: '"' + name + '" restored',
+    });
+
+    renderTrashSources(trashSources);
+  })
+  .catch(function (error) {
```

Comment:
The way `.then()` and `.catch()` are chained to the previous function and indented and so on should be the same

Reply:
@tonytlwu can you please explain more about this comment? Do you mean to move then and catch to event function?

Reply:
@AndrRyaz It's about consistent point of line break and indentation. @upplabs-alex-levchenko @sofiiakvasnevska Please make sure new developers understand this.

**Don't**

```js
doSomething().then(function () {
  // Function
})
.catch(function () {
  // Function
});

doSomething()
.then(function () {
  // Function
}).catch(function () {
  // Function
});

doSomething()
.then(function () {
  // Function
})
  .catch(function () {
    // Function
  });
```

**Do**

```js
doSomething().then(function () {
  // Function
}).catch(function () {
  // Function
});

doSomething()
  .then(function () {
    // Function
  })
  .catch(function () {
    // Function
  });
```